controller/DeliveryServer.php

```diff
@@ -55,6 +60,36 @@ public function awaitingAuthorization()
         $this->setData('cancelUrl', _url('cancelExecution', 'DeliveryServer', 'ltiProctoring', ['deliveryExecution' => $deliveryExecution->getIdentifier()]));
     }
 
+    /**
+     * @throws LtiException
+     * @throws common_Exception
+     * @throws common_exception_Error
+     * @throws common_exception_NotFound
+     * @throws common_exception_Unauthorized
+     */
+    public function runDeliveryExecution()
+    {
+        $deliveryExecution = $this->getCurrentDeliveryExecution();
+
+        // Sets the deliveryId to session.
```

Comment:
Comment not really useful, it's clear what the code is doing and comment is not explaining why it has to be done.

Reply:
I believe that I've just copied that :)